docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -47,11 +51,19 @@ To delete existing identity from filesystem realm:
 
 When creating filesystem realm with Elytron API, you can specify `name rewriter` in the constructor. The name rewriter will be applied to identity names to transform them from one form to another. It can be used to normalize the case, trim extra whitespaces, map one naming scheme to another, remove realm component from identity name (e.g. "user@realm" to "user") or to perform validation step on the syntax of the name.
 
-Class `org.wildfly.security.auth.realm.FileSystemSecurityRealm` is used to instantiate the realm.
+To encrypt the filesystem realm programmatically, you can specify a SecretKey when instantiating the realm.
+
+Class `org.wildfly.security.auth.realm.FileSystemSecurityRealm` is used to instantiate the realm. A builder FileSystemSecurityRealm builder is available at `org.wildfly.security.auth.realm.FileSystemSecurityRealmBuilder` to make the instantiating process easier.
```

Comment:
Maybe we could reword this to something like:

A `org.wildfly.security.auth.realm.FileSystemSecurityRealm` can be instantiated using a `builder()` method as shown in the example below.